.github/workflows/python-package.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        include:
+          # Python 3.4 is not available on mac and windows so it needs to to
+          # be manually added to the matrix for linux.
+          - python-version: 3.4
+            os: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install codecov
+        python scripts/ci/install
+        python scripts/ci/install-check
+    - name: Run tests and checks
+      run: |
+        python scripts/ci/run-tests
+        python scripts/ci/run-check
+
+    - name: codecov
```

Comment:
Codecov does not seem to be appearing on this PR. I wonder if we are missing anything here? It might be worth trying the codecov action: https://github.com/codecov/codecov-action

Reply:
Its in the list of checks. The checks list is large now due to windows/mac. The comment did appear in my fork, don't know why its not here.